WordPress/src/main/java/org/wordpress/android/ui/media/services/MediaUploadService.java

```diff
@@ -360,4 +361,19 @@ private boolean mediaAlreadyQueuedOrUploading(MediaModel mediaModel) {
         }
         return false;
     }
-}
\ No newline at end of file
+
+    public static boolean hasPendingMediaUploadsForPost(PostModel postModel) {
```

Comment:
can we pass the postID here instead of the Model? given that's the only thing being accessed within this method after all